modules/consensus/accept_test.go

```diff
@@ -321,13 +321,13 @@ func TestInconsistentCheck(t *testing.T) {
 	// Corrupt the consensus set.
 	var sfod types.SiafundOutputID
 	var sfo types.SiafundOutput
-	for id, output := range cst.cs.siafundOutputs {
+	cst.cs.db.forEachSiafundOutputs(func(id types.SiafundOutputID, output types.SiafundOutput) {
 		sfod = id
 		sfo = output
-		break
-	}
+	})
```

Comment:
All you need to do is pull a single element out of the loop lol.

Perhaps there's a better way to do that than forEach?